src/navigation/read_objects.cc

```diff
@@ -23,9 +23,9 @@ readObjects(const filesystem::path &objectFilePath)
 
     cv::FileStorage fs(objectFilePath.str(), cv::FileStorage::READ);
     std::vector<double> vertex(2);
-    for (const auto &objectNode : fs["objects"]) {
+    for (const auto objectNode : fs["objects"]) {
```

Comment:
is this a bad merge or did it upset clang somehow?

Reply:
Clang helpfully pointed out that ``fs["objects"]`` doesn't actually return a reference so it was actually correct before I changed it in the last PR :stuck_out_tongue: